content/pt-br/docs/reference/using-api/client-libraries.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Bibliotecas de clientes
+content_type: conceito
+weight: 30
+---
+
+<!-- overview -->
+Esta página contém uma visão geral das bibliotecas clientes para usar a API do Kubernetes por meio de várias linguagens de programação.
+
+
+<!-- body -->
+Para desenvolver aplicações usando a [API REST do Kubernetes](/docs/reference/using-api/),
+você não precisa implementar as chamadas da API e os tipos de solicitação/resposta. 
+Você pode usar uma biblioteca cliente para a linguagem de programação que está usando.
+
+As bibliotecas clientes geralmente lidam com tarefas comuns para você, como autenticação.
+A maioria das bibliotecas de clientes podem fazer a descoberta e usar a Conta de Serviço do Kubernetes para
```

Comment:
Suggestion: "Conta de Serviço" does not directly refer to a Kubernetes resource (that would be `ServiceAccount` instead). In this case, I think it would read better without capital letters at the beginning of each word.